src/codehelp/formatter.vala

```diff
@@ -232,7 +240,7 @@ namespace Vls.Formatter {
         conf["nl_after_func_body_class"] = "2";
         conf["eat_blanks_before_close_brace"] = "true";
         conf["pp_indent_count"] = "0";
-        string[] args = {"uncrustify", "-c", "-", "-l", "vala"};
+        string[] args = { "uncrustify", "-c", "-", "-l", "vala", "-q" };
```

Comment:
`-q` is quiet mode, so won't this mean that `stderr_buf` will always be empty/null?

Reply:
I had the problem that it output something like "main(977): Parsing: 11270 bytes (11270 chars) from stdin as language VALA\n" to stderr and the if condition then simply failed

Reply:
Maybe we could check whether stderr is only one line and simply matches "....Parsing.....(.... chars) from stdin as language VALA)"

Reply:
Ah, I see. Then you can simply check the return code rather than if `stderr_buf != null`.